webserver/vhs-stormforger/README.md

```diff
@@ -0,0 +1,112 @@
+# VHS to Stormforger Demonstration
+
+This example demonstrates a workflow optimizing a web application using a load test that is automatically generated from recorded HTTP traffic. This HTTP traffic is captured in the application to be optimized using [VHS](https://github.com/rename-this/vhs), the load test is generated using StormForge Performance Testing, and the application is optimized using StormForge Optimize. This workflow has three main phases:
+
+1. Record HTTP traffic using VHS.
+2. Configure load test generation container and Optimize experiment.
+3. Run Optimize experiment with load test.
+
+## Capture Data with VHS
+
+The rest of this writeup will assume that you have recorded some network traffic with VHS, and that the recorded network traffic is stored in a Google Cloud Storage object.
+This network traffic should be captured in an environment that is similar or identical to the application that will be optimized.
+The [VHS repository](https://github.com/rename-this/vhs) contains a demonstration that shows how to use VHS to capture http traffic in a kubernetes cluster running the voting web app.
+It can be find in the subdirectory `[hack/demo](https://github.com/rename-this/vhs/tree/main/hack/demo)`.
+This demo deploys the voting web app with a VHS container in each of voting-service pod (4 pods), generates some load with simple script using `curl`, and captures 5 minutes of load to a Google Cloud Storage bucket.
+
+
+### Notes:
+
+- There is a `vhsdemo` service account on GCS that can be used for this demo, or a new one can be created.
```

Comment:
That seems to be internal information - not really helpful for the general if we already omit a concrete example with our own bucket names etc.